### ISSUE TITLE
Intergration of plenv-contrib

### ISSUE THREAD

**Issue by hurricup** [miyagawa/plenv-contrib#12]
I've got a request of support of `plenv-contrib` in the Perl5 plugin for IntelliJ IDEA: https://github.com/Camelcade/Perl5-IDEA/issues/2054

However, it seems that there is only one way to use created library: `plenv use version@lib`

And it's not possible to use it using something like `PLENV_LIBRARY` environment variable with `plenv exec` or `plenv exec-all --with version@lib ...`

Would be nice to have one.

**Comment by miyagawa** [comment]
If you look at the implementation of `plenv-use` https://github.com/miyagawa/plenv-contrib/blob/master/bin/plenv-use all it does is to build up environment variables for a new shell using local::lib and plenv and to launch a new shell with it. There's no reason you can't do it on your own.

**Comment by hurricup** [comment]
Yes, I understand that. But with external tool I believe that proper way is to use one, not to re-implement it's logic.
